pkg/edgeview/src/basics.go

```diff
@@ -236,15 +236,29 @@ func getAddrFromJWT(token string, isServer bool, instID int) (string, string, er
 		}
 	}
 
-	if strings.Contains(jdata.Dep, "/") {
-		urls := strings.SplitN(jdata.Dep, "/", 2)
+	// remove the https:// prefix if exists
+	jdataDep := jdata.Dep
+	if strings.HasPrefix(jdataDep, "https://") {
```

Comment:
Simpler to just call strings.TrimPrefix("https://") followed by strings.TrimPrefix("http://") to make sure they are not present.

Reply:
good point. done.